### PR TITLE
Password field hide

### PR DESCRIPTION
**Pull request by woodward4422**
What this PR is for: 
1.  Hides the password field when the user is signing in with the google login 
2. Makes sure validation works for both scenarios (not provider auth and provider auth)

### CODE REVIEW

**Comment by LinnierGames** [comment]
[Remove Password Field in RegisterVc](https://trello.com/c/geyOp7Vi/153-remove-password-field-in-registervc)

File: KifuSF/DonationRequirementsService + Moya.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    
+    var task: Task {
+        return .requestPlain
+    }
+    
+    var headers: [String : String]? {
+        return nil
+    }
+    
+
```
Comment (pondorasti) [inline KifuSF/DonationRequirementsService + Moya.swift:46]: Nit: empty line

File: KifuSF/DonationRequirementsService.swift [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                completion(dataString)
+                
+            case .failure(let error):
+                assertionFailure("something went wrong \(error.localizedDescription)")
+                return completion(nil)
+            }
+        }
+    }
+
+
```
Comment (pondorasti) [inline KifuSF/DonationRequirementsService.swift:32]: Nit: empty lines

**Comment by pondorasti** [comment]
General feeling: looks good, just add the missing marks

**Review by LinnierGames — APPROVED** [review]
do to time, I'm approving this PR

**Pull request merged**